### PR TITLE
Enable docs-no-retest plugin on installer repo

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

File: cluster/ci/config/prow/plugins.yaml [hunk 1]
```diff
@@ -277,6 +277,7 @@ plugins:
   - trigger
 
   openshift/installer:
+  - docs-no-retest
```
Comment (kargakis) [inline cluster/ci/config/prow/plugins.yaml]: Doesn't add any value now that we have tide

Comment (crawford) [reply cluster/ci/config/prow/plugins.yaml]: Why is that? (Sorry, I'm new to these tools)

**Comment by kargakis** [comment]
Tide does not retest a PR that has been tested on top of current HEAD. The
submit queue used to retest every PR twice and we used docs-no-retest to
make it avoid the second retest for documentation fixes. Tide has replaced
the submit queue.

**Comment by crawford** [comment]
Ah, so that label only affects the merge queue? That would explain why CI is still running end-to-end tests on simple documentation fixes. It sounds like we need to use a regular expression to limit when PR tests are run. Of course, tide will still run tests on its batches, but that's okay.

**Pull request merged**